src/components/BarStack/index.tsx

```diff
@@ -1,37 +1,55 @@
 import styled from '@emotion/styled'
 import { darken, lighten, transparentize } from 'polished'
-import React, { MouseEventHandler, useMemo } from 'react'
+import React, { MouseEventHandler, ReactNode, useMemo } from 'react'
+import { Tooltip } from '..'
```

Comment:
`Tooltip` should be imported from `../Tooltip` to avoid the "Dependency cycle detected" error